docs/request-body-must-exist-for-put-patch.md

```diff
@@ -0,0 +1,102 @@
+# RequestBodyMustExistForPutPatch
+
+## Category
+
+ARM Error
+
+## Applies to
+
+ARM OpenAPI(swagger) specs
+
+
+## Related ARM Guideline Code
+
+- RPC-Put-V1-28, RPC-Patch-V1-12
+
+## Output Message
+
+A Put or Patch request must always have a request body defined. 
+
+## Description
+This rule applies for tracked as well as proxy resources. This applies even in the case where there are no properties defined in the properties bag. 
+
+
+## Why the rule is important
+
+Making the request body mandatory for the Put or Patch request is to ensure that the output of the Get request may be passed in to the Put request and that can be achieved only if the Put has a request body defined. For Patch, it does not make sense to patch an empty payload.
```

Comment:
> Put or Patc [](http://example.com/codeflow?start=41&length=12)

Is it PUT or PATCH?
or PUT and PATCH

Reply:
This sentence should only have Put specified. Will correct it